app/controllers/application_controller.rb

```diff
@@ -36,5 +50,10 @@ def authenticate_admin!
 
   def authorize
     redirect_to root_path, alert: 'Access denied' unless can? :manage, @question
+  end 
+
```

Comment:
I know you didn't write the 'authorize' method, but it doesn't seem to relate to this project at all and should probably be deleted.